### DISCUSSION TITLE
How to run EAM calculations

### DISCUSSION THREAD

**Discussion by mcaroba** [libAtoms/QUIP#262] [Q&A] [answered]
Is there any online (or offline) documentation on how to run embedded-atom method (EAM) calculations with QUIP? The docs say this is possible (it's actually the first item in the list of featured methods). How is it invoked, how to find potential files, etc?

**Comment by gabor1** [comment, marked as answer]
it's called "Glue" model. you can see how it's used in the silicon PRX potential. the glue.xml has three parts,  a pair potential, and a radial density, and an embedding function (energy as a function of density). for the si prx only the pair potential is used, the rest set to zero, but you can see the format in the file.

**Reply by mcaroba** [reply]
OK, thanks. I have dealt with glue potentials in the past, but just those with the pair term, not the density dependent term. Hopefully I can do the parametrization of that part.

**Comment by gabor1** [comment]
[glue.xml.zip](https://github.com/libAtoms/QUIP/files/5673617/glue.xml.zip)

**Comment by jameskermode** [comment]
Glue != EAM, though

We also have EAM_ErcolAd in QUIP. I wouldn't particularly recommend using it in preference to EAM from ASE, matscipy or LAMMPS, unless you need to combine it with GAP.

**Comment by gabor1** [comment]
I guess one could recreate most EAMs as a glue, no? which EAMs are not Glues?

**Comment by jameskermode** [comment]
I've never taken the time to look at the differences, just pointing out we have both implemented so presumably they're not exactly equivalent.

**Reply by bernstei** [reply]
Both are the sum of a pair potential and a nonlinear ("embedding") function of a spherically symmetric "density" linearly summed over the neighbors.  The differences in the QUIP implementation are in the level of generality that are allowed.  I think the Glue allows arbitrary splined curves, and so can encompass any EAM, but not 100% sure of this.

**Reply by gabor1** [reply]
yes, all three 1d functions are represented as splines in glue

**Comment by jameskermode** [comment]
(I'm going to press 'convert to discussion' on this issue, since it doesn't report a fault in the code)

**Reply by mcaroba** [reply]
Thanks and sorry I put it as an issue, I wasn't aware of these "discussions".

**Reply by jameskermode** [reply]
No worries - this is the very first discussion, we just turned them on today.

**Reply by mcaroba** [reply]
Looks good, like Stack Overflow with unlimited comment space, could get out of hand easily!

**Reply by bernstei** [reply]
> Looks good, like Stack Overflow with unlimited comment space, could get out of hand easily!

Except without scoring

**Reply by mcaroba** [reply]
I can score your comment! You just get paid in thumbs up!
